beta-src/src/components/ui/WDPositionContainer.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import { useMemo } from "react";
+
+export enum Position {
+  BOTTOM_LEFT = "bottom-left",
+  BOTTOM_RIGHT = "bottom-right",
+  TOP_LEFT = "top-left",
+  TOP_RIGHT = "top-right",
+}
```

Comment:
Are these used elsewhere? Should it be placed in the `/enums` folder?

Reply:
its specific to this component and easier to import like this.

```
import WDPositionContainer, {Position} from ...
```

as opposed to: 
```
import WDPositionContainer from ...
import {Position} from ...
```
it would make sense if the Position could be used independently, i agree. but in this case, i don't believe that is a scenario.

Reply:
@shelbycodazen I agree, @nebevets we should move these to the enums folder even though I did like the way we can import multiple from this file.